pkg/build/config/versions.go

```diff
@@ -204,4 +204,31 @@ var Versions = VersionMap{
 			StorybookSrcDir:      "artifacts/storybook",
 		},
 	},
+	CloudMode: {
+		Variants: []Variant{
+			VariantLinuxAmd64Musl,
```

Comment:
Is Musl the right one here? I thought for that target glibc would be the one we need :/

Reply:
We need the musl version for Docker, which will be used in the future for cloud.